packages/mobile/ios/AudiusReactNative/Info.plist

```diff
@@ -42,7 +42,13 @@
 			<string>instagram-stories</string>
 			<string>instagram</string>
 			<string>itms-apps</string>
+      <string>snapchat</string>
 		</array>
+    <key>SCSDKClientId</key>
+    <string>6f90a744-e5d0-4115-ad14-fb1a356085cd</string>
```

Comment:
Is this supposed to be secret? Or is it safe to have in here

Reply:
not secret!